src/localization/dispensingStrings.json

```diff
@@ -1,6 +1,7 @@
 {
   "gb": {
     "vaccination_history": "Vaccination history for",
+    "vaccination_details": "Vaccination Event Details",
```

Comment:
:nit the other strings are only uppercase for the initial word. I wonder if it should be `Vaccination event details` 🤔

Reply:
I was conflicted on this 😆 - lots of page titles (and some other modals, e.g. 'Stocktake Details', 'Requisition Details') have all uppercased first letters, but you're also right that the other strings in this area aren't... I suppose better to keep consistency within the section 😅